provisioning_templates/user_data/kickstart_default_user_data.erb

```diff
@@ -66,5 +66,12 @@ fi
 <%= snippet 'saltstack_setup' %>
 <% end -%>
 
+<% if host_param_true?('ansible_tower_provisioning') -%>
+    <%= save_to_file('/tmp/ansible_provisioning_call.sh', snippet('ansible_tower_callback_script')) %>
+    bash /tmp/ansible_provisioning_call.sh
+<% end -%>
```

Comment:
Can you standardize both here and in the kickstart finish template on calling `/bin/sh`? Bash might not be present on some non-RH images.

In the finish template the script is called directly however I don't understand how this can work since it's being created using `cat` without executable permissions. Weird, that's probably a bug.